backend/db/load.py

```diff
@@ -8,22 +8,26 @@
 from backend.db.utils import get_ddl_statements, check_if_updated, current_datetime, insert_from_dict, \
     is_table_up_to_date, load_csv, refresh_termhub_core_cset_derived_tables, run_sql, get_db_connection, sql_in, \
     sql_query, update_db_status_var
-from enclave_wrangler.config import FAVORITE_DATASETS
+from enclave_wrangler.config import DATASET_REGISTRY
 from enclave_wrangler.datasets import download_datasets
 from enclave_wrangler.objects_api import download_favorite_objects
 
 DB = CONFIG["db"]
 SCHEMA = CONFIG['schema']
 DATASET_TABLES = [
-    'code_sets',
+    # Vocab
     'concept',
     'concept_ancestor',
     'concept_relationship',
+    'relationship',
```

Comment:
@Sigfried This table was missing in `initialize()`. I feel like it should be there.